src/services/document-uploader.ts

```diff
@@ -0,0 +1,18 @@
+export const documentToBase64 = (file: File): Promise<string> =>
```

Comment:
Perhaps this file should also be named `document-to-base-64.ts`?

Reply:
Addressed